libs/portals/admin/ids-admin/src/lib/messages.ts

```diff
@@ -778,4 +778,28 @@ export const m = defineMessages({
     defaultMessage:
       'Should personal representatives automatically get this scope for their clients',
   },
+  successDeletingClient: {
+    id: 'ap.ids-admin:success-deleting-client',
+    defaultMessage: 'Successfully deleted client',
+  },
+  deleteClient: {
+    id: 'ap.ids-admin:delete-client',
+    defaultMessage: 'Delete client in {environment} environment',
+  },
+  delete: {
+    id: 'ap.ids-admin:delete',
+    defaultMessage: 'Delete',
+  },
+  closeDeleteModal: {
+    id: 'ap.ids-admin:close-delete-modal',
+    defaultMessage: 'Close',
+  },
+  deleteClientDescription: {
+    id: 'ap.ids-admin:delete-client-action-card-label',
+    defaultMessage: 'All your apps using this client will stop working.',
```

Comment:
I'm suggesting an update in Figma as we are always talking about application instead of client.
```suggestion
    defaultMessage: 'All your apps using this application ID will stop working.',
```